Source/SPSplitView.m

```diff
@@ -950,6 +950,11 @@ - (NSArray *)_suggestedSizesForTargetSize:(CGFloat)targetSize respectingSpringsA
 		}
 	}
 
+	if (delegate && [delegate respondsToSelector:@selector(allowSplitViewResizing)] && ![delegate allowSplitViewResizing]) {
```

Comment:
I don't think you need the first `delegate` check as it will be false for the rest of the two checks anyways if it's nil :)

Reply:
Won't `[nil respondsToSelector:@selector(allowSplitViewResizing)]` crash?

Reply:
Not in Objective-C :) https://stackoverflow.com/a/2696909/4927614

But it's a nit-pick, so leave it there of course if you want :)

Reply:
Cool, thx for reminding me about that. It's been years since I coded in Objective-C. I'm sure I used that trick in the past, and will in the future.